charts/kargo/templates/controller/cluster-roles.yaml

```diff
@@ -106,6 +98,14 @@ metadata:
     {{- include "kargo.labels" . | nindent 4 }}
     {{- include "kargo.controller.labels" . | nindent 4 }}
 rules:
+- apiGroups:
+  - argoproj.io
+  resources:
+  - analysistemplates
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
These permissions are better off down here where they will not be granted in the event that Rollouts integration is completely disabled.